packages/engine-core/src/data-proxy/utils/request.ts

```diff
@@ -80,13 +75,12 @@ function buildResponse(incomingData: Buffer[], response: IncomingMessage): Reque
  * @param options
  * @returns
  */
-function nodeFetch(url: string, options: RequestOptions = {}): Promise<RequestResponse> {
+async function nodeFetch(url: string, options: RequestOptions = {}): Promise<RequestResponse> {
+  const https: typeof Https = await globalThis[['e', 'v', 'a', 'l'].join('')](`import('https')`)
```

Comment:
Used to obfuscate `eval` from bundlers. Vercel Edge Functions fail on build if any eval is detected, but this part of the code is not reachable at runtime, so it is safe. The purpose of this is for `@prisma/client/edge` to work on Node as well, because `fetch` is only available from Node@18.